### PR TITLE
Bump parent, enable dependabot

### PR DESCRIPTION
**Pull request by timja**
Requires https://github.com/jenkinsci/matrix-project-plugin/pull/75

cc @jglick required for https://github.com/jenkinsci/bom/pull/923

### CODE REVIEW

**Comment by jglick** [comment]
@jtnord any idea if this plugin has a maintainer?

**Comment by timja** [comment]
Lots of people claiming to be maintainers 😉 
https://github.com/jenkins-infra/repository-permissions-updater/blob/master/permissions%2Fplugin-matrix-project.yml#L9-L16

**Comment by jglick** [comment]
I have no interest in maintaining it in terms of features (superseded by Pipeline) but since this is blocking `bom` (and it is a common dependency so really needs to be in `bom`) I will try to push out a release.

File: pom.xml [hunk 1]
```diff
@@ -19,6 +19,7 @@
       <jenkins.version>2.282</jenkins.version>
       <java.level>8</java.level>
       <spotbugs.skip>true</spotbugs.skip> <!-- Clean up redundant null check warnings and re-enable after SECURITY-1339 is released -->
+      <powermock.version>2.0.9</powermock.version>
```
Comment (jglick) [inline pom.xml:22]: Better to rewrite or delete PM-based tests.

**Review by jglick — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (timja) [reply pom.xml:22]: didn't want to waste my time on this repo 😄

Comment (jglick) [reply pom.xml:22]: Not enough it seems:

```
java.lang.NoSuchMethodError: org.mockito.Answers.get()Lorg/mockito/stubbing/Answer;
	at org.powermock.api.extension.listener.AnnotationEnabler.standardInject(AnnotationEnabler.java:85)
	at org.powermock.api.extension.listener.AnnotationEnabler.beforeTestMethod(AnnotationEnabler.java:55)
	at org.powermock.tests.utils.impl.PowerMockTestNotifierImpl.notifyBeforeTestMethod(PowerMockTestNotifierImpl.java:82)
```

Comment (timja) [reply pom.xml:22]: 😢 , can look tomorrow unless you can take a quick look

Comment (jglick) [reply pom.xml:22]: I will try it.

Comment (jglick) [reply pom.xml:22]: https://github.com/jenkinsci/plugin-pom/pull/442

**Pull request merged**